src/layers/conv.jl

```diff
@@ -393,3 +393,49 @@ function Base.show(io::IO, l::EdgeConv)
     print(io, ", aggr=", l.aggr)
     print(io, ")")
 end
+
+
+"""
+GINConv([fg,] nn, [eps])
+
+Graph Isomorphism Network
+
+# Arguments
+
+- `fg`: Optionally pass in a FeaturedGraph as input.
+- `nn`: A neural network/layer.
+- `eps`: Weighting factor. Default 0.
+
+"""
```

Comment:
can you add a reference to the paper in the docstring?

Reply:
Done